src/models/AccordionItem.php

```diff
@@ -16,19 +17,49 @@ class AccordionItem extends DataObject
     private static $db = [
         'Title'     => 'Varchar(255)',
         'Content'   => 'HTMLText',
+        'Group'     => 'Int(1)',
         'SortOrder' => 'Int',
     ];
 
     private static $has_one = [
         'Page' => 'Page'
     ];
 
+    /**
+     * @var array
+     */
+    private static $summary_fields = [
+        'Title' => 'Title',
+        'Group' => 'Accordion ID',
+    ];
+
     private static $default_sort = 'SortOrder ASC';
 
+    /**
+     * @return RequiredFields
+     */
+    public function getCMSValidator()
+    {
+        return new RequiredFields(['Group']);
```

Comment:
I wouldn't make it required, if I don't want to use it, why should I be forced to?